project_classification/project_classification_view.xml

```diff
@@ -46,10 +48,10 @@
       <field name="model">project.project</field>
       <field name="inherit_id" ref="project.edit_project" />
       <field name="arch" type="xml">
-        <xpath expr="/form/sheet/notebook/page[@string='Other Info']/group/group[3]/field[@name='parent_id']" position="replace">
-          <field name="classification_id" widget="selection" on_change="onchange_classification_id(classification_id)"/>
+        <xpath expr="/form/sheet/notebook/page[@string='Other Info']//field[@name='parent_id']" position="replace">
+          <field name="classification_id" required="1" widget="selection" on_change="onchange_classification_id(classification_id)"/>
```

Comment:
@yvaucher yes, but then we are back on the original issue: there is no default classification present to be set on existing projects.

It's probably best to use the pre_init_hook to create a "old project" classification if some exists, and set in on every existing project; I think in that case, the runbot will not be red (as a value was set on existing projects), and the "default" value will not be used for new projects.